api_docs/changelog.md

```diff
@@ -20,6 +20,13 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 263**
+
+* `PATCH /realm`, [`POST /register`](/api/register-queue),
+  [`GET /events`](/api/get-events): Added `can_create_public_channel_group`
+  realm setting, which is either ID of a named user group with the permission,
+  or an object describing the set of users and groups with the permission.
```

Comment:
Are we fine with mentioning "object" since we only support integer IDs for now in production.

Reply:
I think we should just say something like:

> Added `can_create_public_channel_group` realm setting, which is a group-setting value describing the set of users with permission to create channels. 

And maybe `group-setting value` can be a link to the page on those.